cypress/page/shoping-cart.page.ts

```diff
@@ -4,8 +4,8 @@ class ShopingCartPage {
     private goToCheckoutBtn: string;
 
     constructor() {
-        this.goToCartBtn = "[style*='display: block;'] .button-container > a";
-        this.goToCheckoutBtn = ".cart_navigation span";
+        this.goToCartBtn = ".button-container > .button-medium";
```

Comment:
can you use an attribute to indicate we are referring to the add to cart button?